src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Localizations/en.VirtoCommerce.DemoCustomerSegmentsModule.json

```diff
@@ -53,6 +62,12 @@
           "name": "Name"
         }
       }
+    },
+    "dialogs": {
+      "customer-segment-delete": {
+        "title": "Delete confirmation",
+        "message": "Customer {{segments}} will be deleted and the customers of this {{count}} will no longer be receiving the tagged promotions"
```

Comment:
Is this message related to AC?

Reply:
This message is inside the modal when deleting a segment

Reply:
Just split this messages for one segment and several.

Reply:
done